static/index.ejs

```diff
@@ -47,132 +47,354 @@
+          0%,
+          6.382979% {
+            stroke-dashoffset: 1150;
+          }
+          6.382979% {
+            animation-timing-function: ease-out;
+          }
+          43.829787% {
+            stroke-dashoffset: 0;
+            animation-timing-function: ease-out;
+          }
+          99.999% {
+            stroke-dashoffset: -1150;
+          }
+          100% {
+            stroke-dashoffset: 1150;
+          }
+        }
+
+        @keyframes draw_g1_p2 {
+          /* start= 230ms =>  9.787234%, 40% point=45.872340% */
+          0%,
+          9.787234% {
+            stroke-dashoffset: 1150;
+          }
+          9.787234% {
+            animation-timing-function: ease-out;
+          }
+          45.872340% {
+            stroke-dashoffset: 0;
+            animation-timing-function: ease-out;
+          }
+          99.999% {
+            stroke-dashoffset: -1150;
+          }
+          100% {
+            stroke-dashoffset: 1150;
+          }
+        }
+        @keyframes draw_g1_p1 {
+          /* start= 280ms => 11.914894%, 40% point=47.148936% */
+          0%,
+          11.914894% {
+            stroke-dashoffset: 1150;
+          }
+          11.914894% {
+            animation-timing-function: ease-out;
+          }
+          47.148936% {
+            stroke-dashoffset: 0;
+            animation-timing-function: ease-out;
+          }
+          99.999% {
+            stroke-dashoffset: -1150;
+          }
+          100% {
+            stroke-dashoffset: 1150;
+          }
+        }
+        @keyframes draw_g1_p0 {
+          /* start= 330ms => 14.042553%, 40% point=48.425532% */
+          0%,
+          14.042553% {
+            stroke-dashoffset: 1150;
+          }
+          14.042553% {
+            animation-timing-function: ease-out;
+          }
+          48.425532% {
+            stroke-dashoffset: 0;
+            animation-timing-function: ease-out;
+          }
+          99.999% {
+            stroke-dashoffset: -1150;
+          }
+          100% {
+            stroke-dashoffset: 1150;
+          }
+        }
+
+        @keyframes draw_g2_p3 {
+          /* start= 400ms => 17.021277%, 40% point=50.212766% */
+          0%,
+          17.021277% {
+            stroke-dashoffset: 1150;
+          }
+          17.021277% {
+            animation-timing-function: ease-out;
+          }
+          50.212766% {
+            stroke-dashoffset: 0;
+            animation-timing-function: ease-out;
+          }
+          99.999% {
+            stroke-dashoffset: -1150;
+          }
+          100% {
+            stroke-dashoffset: 1150;
+          }
+        }
+        @keyframes draw_g2_p1 {
+          /* start= 500ms => 21.276596%, 40% point=52.765957% */
+          0%,
+          21.276596% {
+            stroke-dashoffset: 1150;
+          }
+          21.276596% {
+            animation-timing-function: ease-out;
+          }
+          52.765957% {
```

Comment:
Lets round all these keyframe values, I don't think we need such surgical precision 😅